pyugrid/ugrid.py

```diff
@@ -38,6 +37,9 @@ class UGrid(object):
     The internal structure mirrors the netcdf data standard.
     """
 
+    # When reading from netCDF, check for Fortran v. C ordering.
+    check_array_order = True
```

Comment:
This is painful and ugly -- when we check array order, all we can do is hope that if one dimension happens to be the same size as the number of nodes, then we have a certain order -- subject to error , particularly on test datasets with tiny grids :-) -- maybe not likely to be an issue with "real" grids, but shouldn't this be addresses in the UGRID spec?

@rsignell-usgs : I think you were the first write this code -- any ideas?